modules/UI/shared_video/SharedVideo.js

```diff
@@ -252,10 +250,10 @@ export default class SharedVideoManager {
             // in onPlayerStateChange
             player.playVideo();
 
-            let thumb = new SharedVideoThumb(
-                self.url, SHARED_VIDEO_CONTAINER_TYPE, VideoLayout);
-            thumb.setDisplayName(player.getVideoData().title);
-            VideoLayout.addRemoteVideoContainer(self.url, thumb);
+            VideoLayout.addSharedVideoThumbnail(
+                self.url,
+                player.getVideoData().title,
+                SHARED_VIDEO_CONTAINER_TYPE);
```

Comment:
Not sure I understand the benefit of moving the addSharedVideoThumbnail to VideoLayout. It's still the responsibility of the SharedVideo.js to add it's thumbnail (there's no event that just tells the VideoLayout), so I feel it doesn't add much. That said, this is a transitionary state, so I don't mind keeping it in the PR.